File: Java Word Examples/DocumentPropertiesApiDemo.java
import spire.cloud.word.sdk.client.*;
import spire.cloud.word.sdk.client.api.DocumentPropertiesApi;
import spire.cloud.word.sdk.client.model.*;

import java.util.*;

public class DocumentPropertiesApiDemo {
    static String appId = "your id";
    static String appKey = "your key";
    static String baseUrl = "https://api.e-iceblue.cn";
    static Configuration wordConfiguration = new Configuration(appId, appKey, baseUrl);
    static DocumentPropertiesApi documentPropertiesApi = new DocumentPropertiesApi(wordConfiguration);

    public static void addCustomDocumentProperties() throws ApiException {
        String name = "addCustomDocumentProperties.docx";
        List<CustomDocumentProperty> properties = new ArrayList<CustomDocumentProperty>();
        properties.add(new CustomDocumentProperty("propertiesName1", "propertiesValue1"));
        properties.add(new CustomDocumentProperty("propertiesName2", "propertiesValue2"));
        String password = null;
        String folder = "input";
        String destFilePath = "output/addCustomDocumentProperties_out.docx";
        String storage = null;
        documentPropertiesApi.addCustomDocumentProperties(name, properties, destFilePath, password, folder, storage);
    }

    public static void deleteCustomDocumentProperty() throws ApiException {
        String name = "deleteCustomDocumentPropertie.docx";
        String propertieName = "E-iceblue";
        String password = null;
        String folder = "input";
        String storage = null;
        String destFilePath = "output/deleteCustomDocumentProperties_output.docx";
        documentPropertiesApi.deleteCustomDocumentProperty(name, propertieName, destFilePath, password, folder, storage);
    }

    public static void getBuiltinDocumentProperties() throws ApiException {
        String name = "getBuiltinDocumentProperties.docx";
        String password = null;
        String folder = "input";
        String storage = null;
        BuiltinDocumentProperties response = documentPropertiesApi.getBuiltinDocumentProperties(name, password, folder, storage);
    }

    public static void getCustomDocumentProperties() throws ApiException {
        String name = "getCustomDocumentProperties.docx";
        String password = null;
        String folder = "input";
        String storage = null;
        List<CustomDocumentProperty> response = documentPropertiesApi.getCustomDocumentProperties(name, password, folder, storage);
    }
}

File: C# Word Examples/DocumentPropertiesApiDemo.cs
using Spire.Cloud.Word.Sdk.Api;
using Spire.Cloud.Word.Sdk.Client;
using Spire.Cloud.Word.Sdk.Model;
using System;
using System.Collections.Generic;

namespace CloudWord
{
    class DocumentPropertiesApiDemo
    {
        static string appId = "your id";
        static string appKey = "your key";
        static string baseUrl = "https://api.e-iceblue.cn";
        static Configuration wordConfiguration = new Configuration(appId, appKey, baseUrl);
        static DocumentPropertiesApi documentPropertiesApi = new DocumentPropertiesApi(wordConfiguration);
        public static void AddCustomDocumentProperties()
        {
            string name = "addCustomDocumentProperties.docx";
            List<CustomDocumentProperty> properties = new List<CustomDocumentProperty>();
            properties.Add(new CustomDocumentProperty("propertiesName1", "propertiesValue1"));
            properties.Add(new CustomDocumentProperty("propertiesName2", "propertiesValue2"));
            string password = null;
            string folder = "input";
            string destFilePath = "output/addCustomDocumentProperties_out.docx";
            string storage = null;
            documentPropertiesApi.AddCustomDocumentProperties(name, properties, destFilePath, password, folder, storage);
        }

        public static void DeleteCustomDocumentProperty()
        {
            string name = "deleteCustomDocumentPropertie.docx";
            string propertieName = "E-iceblue";
            string password = null;
            string folder = "input";
            string storage = null;
            string destFilePath = "output/deleteCustomDocumentProperties_output.docx";
            documentPropertiesApi.DeleteCustomDocumentProperty(name, propertieName, destFilePath, password, folder, storage);
        }

        public static void GetBuiltinDocumentProperties()
        {
            string name = "getBuiltinDocumentProperties.docx";
            string password = null;
            string folder = "input";
            string storage = null;
            var response = documentPropertiesApi.GetBuiltinDocumentProperties(name, password, folder, storage);
        }

        public static void GetCustomDocumentProperties()
        {
            string name = "getCustomDocumentProperties.docx";
            string password = null;
            string folder = "input";
            string storage = null;
            var response = documentPropertiesApi.GetCustomDocumentProperties(name, password, folder, storage);
        }
    }
}
